pkg/mkimage-raw-efi/install

```diff
@@ -70,7 +70,8 @@ root_dev() {
 find_part() {
    PARTS=$(lsblk -anl -o "NAME,PARTLABEL" 2>/dev/null | sed -ne "/$1"'$/s#'"$1##p")
    for p in $PARTS ; do
-      [ -f "/sys/block/$2/$p/dev" ] && echo "$p" && exit 0
+      PART=$(echo "$p"|tr -d '[:space:]')
```

Comment:
why is this needed here @giggsoff ?

Reply:
Because of formatting inside `lsblk`, see result from my PC (spaces between name and partlabel):
```
sda       
sda1      Microsoft reserved partition
sda2      Basic data partition
sda3      
sda5      
nvme0n1   
nvme0n1p1 Basic data partition
nvme0n1p2 EFI system partition
nvme0n1p3 Microsoft reserved partition
nvme0n1p4 Basic data partition
nvme0n1p5 
```

Reply:
Sorry for being dense -- but the first column doesn't seem to ever contain spaces and ```$p``` is iterating over the first column, correct?

Reply:
The problem here in spaces added to $p (something like "sda1      " and "nvme0n1p3 " from my previous sample. Please see new version in last commit, I put changes in top sed.